lib/fathead/python_requests/output.txt

```diff
@@ -0,0 +1,36 @@
+Proxies	A										<p>If you need to use a proxy, you can configure individual requests with the\nproxies argument to any request method:<pre>import requests\n\nproxies = {\n  'http': 'http://10.10.1.10:3128',\n  'https': 'http://10.10.1.10:1080',\n}\n\nrequests.get('http://example.org', proxies=proxies)\n</pre> You can also configure proxies by setting the environment variables\nHTTP_PROXY and HTTPS_PROXY.<pre>$ export HTTP_PROXY="http://10.10.1.10:3128"\n$ export HTTPS_PROXY="http://10.10.1.10:1080"\n\n$ python\n>>> import requests\n>>> requests.get('http://example.org')\n</pre> To use HTTP Basic Auth with your proxy, use the http://user:password@host/ syntax:<pre>proxies = {'http': 'http://user:pass@10.10.1.10:3128/'}\n</pre> To give a proxy for a specific scheme and host, use the\nscheme://hostname form for the key.  This will match for\nany request to the given scheme and exact hostname.<pre>proxies = {'http://10.20.1.128': 'http://10.10.1.10:5323'}\n</pre> Note that proxy URLs must include the scheme.<pre>$ pip install requests[socks]\n</pre></p>	http://docs.python-requests.org/en/master/user/advanced/#proxies
+Raw Response Content	A										<p>In the rare case that you'd like to get the raw socket response from the\nserver, you can access r.raw. If you want to do this, make sure you set\nstream=True in your initial request. Once you do, you can do this:<pre>>>> r = requests.get('https://api.github.com/events', stream=True)\n\n>>> r.raw\n<requests.packages.urllib3.response.HTTPResponse object at 0x101194810>\n\n>>> r.raw.read(10)\n'\x1f\x8b\x08\x00\x00\x00\x00\x00\x00\x03'\n</pre> In general, however, you should use a pattern like this to save what is being\nstreamed to a file:<pre>with open(filename, 'wb') as fd:\n    for chunk in r.iter_content(chunk_size):\n        fd.write(chunk)\n</pre> Using Response.iter_content will handle a lot of what you would otherwise\nhave to handle when using Response.raw directly. When streaming a\ndownload, the above is the preferred and recommended way to retrieve the\ncontent.</p>	http://docs.python-requests.org/en/master/user/quickstart/#raw-response-content
```

Comment:
We need to escape the unicode strings here:

e.g. `\\x1f\\x8b`

Currently they create problems with our DB. We're working to figure out if we can better handle this internally.

It looks like a regex will work here to replace `\x` with `\\x`